src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -9,13 +10,59 @@ type DelegationNodeInfoResponse = {
   delegatedStakeAmount: string;
   networkPercentage?: string;
   commission: number | undefined;
+  delegatorBalance: string;
 };
 
 type DelegationNodeInfoProps = {
   validatorAddress: Types.Address;
   validator: ValidatorData | Validator;
 };
 
+export interface DelegatedStakingActivity {
+  amount: number;
+  delegator_address: Types.Address;
+  event_index: number;
+  event_type: string;
+  pool_address: Types.Address;
+  transaction_version: bigint;
+}
+
+const NUMBER_OF_DELEGATORS_QUERY = gql`
+  query numberOfDelegatorsQuery($poolAddress: String) {
+    current_delegator_balances_aggregate(
+      where: {
+        pool_type: {_eq: "active_shares"}
+        pool_address: {_eq: $poolAddress}
+        amount: {_gt: "0"}
+      }
+      distinct_on: delegator_address
+    ) {
+      aggregate {
+        count
+      }
+    }
+  }
+`;
+
+function useGetNumberOfDelegators(poolAddress: Types.Address) {
```

Comment:
why not just export it a separate hook and use it directly? adding this as part of delegation info will introduce unnecessary dependency.